tasks/collect-data/README.md

```diff
@@ -21,11 +21,14 @@ should not be present in the Release data section).
 | Name                 | Description                                        | Optional | Default value |
 |----------------------|----------------------------------------------------|----------|---------------|
 | release              | Namespaced name of the Release                     | No       | -             |
-| releaseplan          | Namespaced name of the ReleasePlan                 | No       | -             |
-| releaseplanadmission | Namespaced name of the ReleasePlanAdmission        | No       | -             |
+| releasePlan          | Namespaced name of the ReleasePlan                 | No       | -             |
+| releasePlanAdmission | Namespaced name of the ReleasePlanAdmission        | No       | -             |
 | snapshot             | Namespaced name of the Snapshot                    | No       | -             |
 | subdirectory         | Subdirectory inside the workspace to be used.      | Yes      | -             |
 
+## Changes in 3.0.0
+* Parameters supplied by the Release Service operator now use camelCase format
```

Comment:
Maybe nitpicking, but this change is not needed because of the Release Service operator change, but because you changed the parameters used when calling this task in all the pipelines. So maybe something like this:
```
* Parameters supplied by the Release Service operator now use camelCase format and for consistency, the `collect-data` task also switched to use camelCase in the parameters.
```

Reply:
+1